### PR TITLE
Field picture question

### LINKED ISSUE #15: Creating a quiz question type "picture" change form field to highlight it is a url

### PR DESCRIPTION
**Pull request by jesus-s-sanzo**
closes #15 
Conditionally shows a text field or text area and appropriate labels depending on the question type.

For some reason the readme.md file comes as changed despite I made no changes on it other than updating my local version with the codemortals master

### CODE REVIEW

**Comment by stemount** [comment]
@jesus-s-sanzo I think you have a case insensitive filesystem or something because README.md already exists and you have readme.md if your PR so I am wondering if that's causing your isues with README.me seeming changed but when on GitHub it is case sensitive?

**Comment by jesus-s-sanzo** [comment]
I use Windows. I think I created and pushed the original readme.md
(lowercaps), that was probably renamed to uppercase, and something weird
happened when I pulled and merged the updated master branch. I'm not sure
how to get rid of it, but I'll try.

I hope is fixed now

File: hosting/src/app/games/quiz/lobby/lobby.component.html [hunk 1]
```diff
@@ -11,7 +11,11 @@ <h2>Create Question</h2>
 
                         <brand-dropdown label="Type" placeholder="Select an option" formControlName="type" [options]="availableTypes"></brand-dropdown>
 
-                        <brand-input label="Question" type="textarea" formControlName="text"></brand-input>
+                        <span *ngIf="questionForm.get('type').value.id =='image'; else standardQuestion">
+                            <brand-label description="(image should include any text for the question)">Image URL</brand-label>
+                            <brand-input   type="text" formControlName="text" placeholder="http://url-of-the-image"></brand-input>
+                        </span>
+                        <ng-template #standardQuestion><brand-input  label="Question" type="textarea" formControlName="text"></brand-input></ng-template>
```
Comment (eddiejaoude) [inline hosting/src/app/games/quiz/lobby/lobby.component.html:18]: Another alternative could be to use the form type attribute as the input type, for pseudo  example

```typescript
 [type]="getType()"
```

**Review by eddiejaoude — APPROVED** [review]
Comment just for information, this area will change when new question types are added

**Pull request merged**